app/src/Controller/ApiController.php

```diff
@@ -56,44 +65,46 @@ public function game(GameContext $context) {
 
       return new JsonResponse([
         'id' => $game->getId(),
+        'isMultiplayer' => $game->getIsMultiplayer(),
         'currentPlayer' => $player,
         'otherPlayer' => $otherPlayer,
         'winner' => $game->getWinner()
       ]);
     }
     return new JsonResponse([
       'id' => $game->getId(),
+      'isMultiplayer' => $game->getIsMultiplayer(),
       'currentPlayer' => $player,
       'winner' => $game->getWinner()
     ]);
   }
 
-  public function cancel(GameContext $context) {
+  public function cancel(Request $request, GameContext $context) {
+    if ($request->getMethod() == 'OPTIONS') {
+      return new Response(Response::HTTP_OK);
+    }
+
     if ($context->getIsPlayer()) {
       $this->em->remove($context->getGame());
       $this->em->flush();
 
       return new Response(
-        'Content',
-        Response::HTTP_OK,
-        ['content-type' => 'text/html']
+        Response::HTTP_OK
```

Comment:
The first arg is the content (the second is the status code), you send "200" in the body here